composer.json

```diff
@@ -31,8 +28,6 @@
         "php": ">=7.0"
     },
     "require-dev": {
-        "phpunit/phpunit": "~6.5.5",
-        "brain/monkey": "dev-master",
```

Comment:
I would like to keep them because `widoz/project-tests-helper` could be changed or removed in future and since I'm using directly those libraries would be better to keep them.

Reply:
Done.